ang/civicase/Dashboard.html

```diff
@@ -0,0 +1,149 @@
+<div id="bootstrap-theme" class="civicase__container">
+  <div class="civicase__dashboard">
+    <h1 class="hidden" crm-page-title crm-document-title="myCasesOnly ? ts('CiviCase Dashboard - My Cases') : ts('CiviCase Dashboard - All Cases')"></h1>
+
+    <div class="civicase__dashboard__tab-container clearfix">
+      <uib-tabset
+        civicase-dashboard-tabset-affix active="activeTab" type="pills"
+        uib-tabset-class="nav-pills-horizontal nav-pills-horizontal-primary clearfix">
+        <!-- Dashboard Tab -->
+        <uib-tab index="0" heading="{{ ts('Dashboard') }}" select="navTabs()">
+
+          <div class="panel panel-default" ng-include="'~/civicase/DashboardSummary.html'"></div>
+
+          <div class="row case-stats-wrapper">
+            <!-- Mockup proscribes different coloring for these panels, but don't see a suitable class in the style guide -->
+            <div class="col-md-4">
+              <div class="panel panel-default">
+                <div class="panel-heading">
+                  <h3 class="panel-title">{{ts('Open Cases')}}</h3>
+                </div>
+                <div class="panel-body"><div civicase-type-donut-chart="{my_cases: myCasesOnly, 'status_id.grouping': 'Opened'}"></div></div>
+              </div>
+            </div>
+            <div class="col-md-4">
+              <div class="panel panel-default">
+                <div class="panel-heading">
+                  <h3 class="panel-title">{{ts('Cases Closed')}}</h3>
+                </div>
+                <div class="panel-body"><div civicase-type-donut-chart="{my_cases: myCasesOnly, 'status_id.grouping': 'Closed'}"></div></div>
+              </div>
+            </div>
+            <div class="col-md-4">
+              <div class="panel panel-default">
+                <div class="panel-heading">
+                  <h3 class="panel-title">{{ts('Average Duration')}}</h3>
+                </div>
+                <div class="panel-body"><div civicase-type-duration-chart="{my_cases: myCasesOnly}"></div>
```

Comment:
Fix indentation
```
<div class="panel-body">
  <div civicase-type-duration-chart="{my_cases: myCasesOnly}"></div>
</div>
```

Reply:
The whole file has been renamed, thats why everything is showing as changed.
But this HTML is part of Dashboard tab, which is not touched in this PR.

It needs to be taken care of when we work on the Dashboard tab ticket.